code/game/objects/items/robot/robot_parts.dm

```diff
@@ -62,6 +62,59 @@
 				return 1
 	return 0
 
+/obj/item/robot_suit/wrench_act(mob/living/user, obj/item/I) //Deconstucts empty borg shell. Flashes remain unbroken because they haven't been used yet
+	var/turf/T = get_turf(src)
+	forceMove(T)
+	if(l_leg || r_leg || chest || l_arm || r_arm || head)
+		if(I.use_tool(src, user, 5, volume=50))
+			if(l_leg)
+				l_leg.forceMove(T)
+				l_leg = null
+			if(r_leg)
+				r_leg.forceMove(T)
+				r_leg = null
+			if(chest)
+				if (chest.cell) //Sanity check.
+					chest.cell.forceMove(T)
+					chest.cell = null
+				chest.forceMove(T)
+				new /obj/item/stack/cable_coil(T, 1)
+				chest.wired = 0

```

Comment:
FALSE

Reply:
What, specifically, is wrong with the code here? Aside from the interlaced `if` statements, it's the code used to deconstruct a functional borg.

Reply:
the 0 needs to be `FALSE`

Reply:
Zero has been replaced with `FALSE`.